index.html

```diff
@@ -1348,6 +1348,19 @@ <h2>
 MAY be encoded in JSON Web Key (JWK) format using the <code>publicKeyJwk</code>
 property. Both the raw 32-byte base58-btc format and JWK format MUST be
 supported by DID Document Processors. The use of other key formats for RSA MUST
+cause the DID Document Processor to raise an error.
+            </td>
+          </tr>
+          <tr>
+            <td>
+ed25519
+            </td>
+            <td>
+ed25519 key values MAY be expressed as the raw 32-byte value encoded in
+base58-btc format using the <code>publicKeyBase58</code> property or
+MAY be encoded in JSON Web Key (JWK) format using the <code>publicKeyJwk</code>
```

Comment:
One question here is ... does anyone really use JWK for ed25519? If not, we could eliminate it and just have the common base58 representation.

Reply:
Yes, use of JWKs for Ed25519 is common. That's why members of the community developed RFC 8037 - to define how to use the 25519 and 448 curves with JSON.

Reply:
I'm not sure if anyone is currently using JWKs as a representation right now for ed25519 keys, (legacy implementations appear to use base58) but I read this as implementers of future DID registries are encouraged to use JWKs rather than base58.

Reply:
@selfissued wrote:
> Yes, use of JWKs for Ed25519 is common.

Do you have data or a citation for this assertion? I expect that's true in certain communities, but which ones, and which implementations?

Reply:
I'm aware of all these implementations:

- python: https://github.com/latchset/jwcrypto/
- node.js: https://github.com/panva/jose
- elixir: https://github.com/joken-elixir/joken
- go: https://github.com/pascaldekloe/jwt
- php: https://github.com/web-token/jwt-framework

I'm sure there are others...